flow-typed/react-flip-move_v2.9.x.js

```diff
@@ -77,11 +70,16 @@ declare module 'react-flip-move' {
     style?: ReactStyles,
   };
 
-  declare export type FlipMoveProps = FlipMoveDefaultProps & Hooks & DelegatedProps & {
-    children?: mixed,
+  declare export type FlipMoveProps = CommonProps & DelegatedProps & {
+    duration: string | number,
+    delay: string | number,
+    staggerDurationBy: string | number,
+    staggerDelayBy: string | number,
+    enterAnimation: AnimationProp,
+    leaveAnimation: AnimationProp,
     appearAnimation?: AnimationProp,
     disableAnimations?: boolean, // deprecated, use disableAllAnimations instead
   };
 
-  declare export default Class<React$Component<FlipMoveDefaultProps, FlipMoveProps, void>>;
+  declare export default ComponentType<FlipMoveProps>;
```

Comment:
😍

Reply:
Seems like we have to revert this part for now =( https://github.com/flowtype/flow-typed/issues/1209